6edac39a68de7552af1d366c9162ac9f4b790b6da0c319f0130a4487f13cce1e.json

```diff
@@ -0,0 +1 @@
+{"untrusted_raw_value":"p2F2AWN1cmx4GGh0dHBzOi8vZmVlbC1taW5pbmcuY29tL2RuYW1la0ZlZWwtTWluaW5nZWVtYWlseB9uaWNvbGFzLm1ldHpnZXJAZmVlbC1taW5pbmcuY29tZnNlcmlhbAJna2V5YmFzZWpmZWVsbWluaW5nZ3R3aXR0ZXJqRmVlbE1pbmluZw==","signature":{"public_key":"btrDmmjedVKvHTZskWKsn0t5C22gwxnwEwpEh/E8zh4=","signature":"AhK8QdZa/Ma9SqN42sRiLlUwo5/0syVgroWlOe7xw2F8pm08JdKJkg1uO7rtum2YXjJIV5d8lGac7ajFxHV6BQ=="}}
```

Comment:
The statement file should be in the `registry/entity` subdirectory but you seem to have put it in the repository root.

Reply:
hello i just made the modification sorry i am not yet very used to use github